package.json

```diff
@@ -1,24 +1,26 @@
 {
-  "name": "gulp-jsdoc3",
-  "version": "0.2.1",
+  "name": "@sugarcrm/gulp-jsdoc3",
+  "version": "0.2.4",
```

Comment:
Can we use latest version gulp-jsdoc3 https://github.com/mlucool/gulp-jsdoc3/blob/master/package.json#L3